src/wrapped/wrappedxml2_private.h

```diff
@@ -250,7 +250,7 @@ GO(xmlCtxtGetLastError, pFp)
 GO(xmlCtxtReadDoc, pFppppi)
 GO(xmlCtxtReadFd, pFpippi)
 GO(xmlCtxtReadFile, pFpppi)
-//GOM(xmlCtxtReadIO, pFEpBBpppi)
+GO(xmlCtxtReadIO, pFppppppi)
```

Comment:
the `B` in the signature means callBacj, so they need special care with a `find_XXXX` and all.